Packs/MicrosoftGraphApplications/Integrations/MicrosoftGraphApplications/MicrosoftGraphApplications_description.md

```diff
@@ -17,5 +26,18 @@ In order to use the Cortex XSOAR Azure application, use the default application
 
 To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal, with mobile and desktop flows enabled.
 
+### Client Credentials Flow
```

Comment:
Lets put this pare above the device code, as this is the recommended authentication method